bsc-plugin/src/lib/rooibos/Utils.ts

```diff
@@ -1,147 +1,54 @@
-import type { BrsFile, ClassStatement, Expression, FunctionStatement, Statement, AnnotationExpression } from 'brighterscript';
-import { createVariableExpression } from 'brighterscript';
+import type { BrsFile, ClassStatement, Expression, FunctionStatement, Statement, AnnotationExpression, AstEditor } from 'brighterscript';
```

Comment:
Removed MANY completely unused methods from here. Can you verify that you don't utilize those functions in some other downstream project?

Reply:
don't use them donwstream :)

Reply:
Great!